python/setup.py

```diff
@@ -30,6 +30,7 @@
                       'boto3',
                       'fastavro',
                       'mmh3',
+                      'moz_sql_parser',
```

Comment:
You also need to add `pyparsing` here.

```suggestion
                      'moz_sql_parser',
                      'pyparsing',
```

Reply:
@xhochy updated the commit. thanks!